apps-rendering/src/components/editions/standfirst/index.tsx

```diff
@@ -26,29 +29,31 @@ const interviewStyles = css`
 	${sidePadding}
 `;
 const galleryStyles = css`
-	${body.medium({ lineHeight: 'tight', fontWeight: 'bold' })}
+	${textEgyptianBold17};
+	line-height: 1.15;
 	${from.mobileMedium} {
-		${body.medium({ lineHeight: 'tight' })}
+		${textEgyptian17};
+		line-height: 1.15;
```

Comment:
We don't need this line. `line-height` has the same value outside of the media query

Reply:
Unfortunately it's still needed as the `textEgyptian17` preset will reset `line-height` back to 1.3 so has to be overridden again.